README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
+start checking your commits with a linter; but you don't want to rewrite
+the entire repository, since it will mess up your history and the
+ability to `git bisect`?
+
+**Difflint** will check only the changes that you make in each commit.
+It's suitable for using as a Git pre-commit hook.
+
+## Linters supported ##
+
+- [JSCS](http://jscs.info) (JavaScript)
+- [JSHint](http://jshint.com/docs) (JavaScript)
+
+## Requirements ##
+
+- Python 3.3 or later
+- JSCS
+- JSHint
+- NPM for installing JSCS and JSHint
+
+## Installation ##
+
+- Clone this repository, and run `python3 setup.py install`.
```

Comment:
I turned the last step into a separate "Configuration (optional)" section so that the installation process looks less daunting.

I also added a sentence with a link to Git's documentation about templates. It's really a Git feature, so I don't think we should re-document it here.

There are several places that a user might keep their Git template; there are four ways to specify a template location to `git init`, so I don't think we should be in the business of trying to autodetect which template a user might want to add Difflint to. A user might even have more than one git template for different contexts.